src/engraving/libmscore/segment.cpp

```diff
@@ -1711,7 +1711,7 @@ Spanner* Segment::firstSpanner(staff_idx_t activeStaff)
                 continue;
             }
             if (s->startSegment() == this) {
-                if (e->staffIdx() == activeStaff || (e->isMeasure() && activeStaff == 0)) {
+                if (e->staffIdx() == activeStaff || activeStaff == 0) {
```

Comment:
Debugged a little bit
The problem here is in the first condition - the track of the segment is invalid, and because of this the check of the staff does not pass.
We need to investigate why the track is invalid.